packages/suite-data/files/bin/bridge/update.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+
+# Fill in your path here:
+BRIDGE_DIR="/Users/XXX/go/src/github.com/trezor/trezord-go"
```

Comment:
We could of course do this the proper way by providing the path through a CLI argument but as we run this once a year it really seems as unnecessary.

Reply:
👌